readme.md

```diff
@@ -62,16 +62,16 @@ body{
 All the steps for one way of doing the above have been written below, but in the wrong order. Put them in the correct order.
 
 ```
-$ touch README.md
-$ git push origin master
-$ git push origin master
-$ git remote add origin git@github.com/username/project-repo.git
-$ git remote add juan git@github.com/juan/project-repo.git
-$ git commit -m "initial commit"
 $ cd project-repo
+$ touch README.md
 $ git init project-repo
+$ git remote add origin git@github.com/username/project-repo.git
 $ git add .
+ git commit -m "initial commit"
+$ git push origin master
+$ git remote add juan git@github.com/juan/project-repo.git
 $ git merge juan/feature
+$ git push origin master
 ```
 
```

Comment:
`git init project-repo` will actually create a local repo, so you want to do that first and then `cd` into it. From there, however, the rest of your git commands look good.